.browserslistrc

```diff
@@ -0,0 +1,5 @@
+edge >= 83
+firefox >= 77
+chrome >= 83
+safari >= 11
+node 10.0
```

Comment:
It looks like another advantage of not using TypeScipt to build the source but Babel. They don't seem to support browser target https://github.com/microsoft/TypeScript/issues/19183.